packages/amplify_datastore/ios/amplify_datastore.podspec

```diff
@@ -17,7 +17,6 @@ The DataStore module for Amplify Flutter.
   s.dependency 'Flutter'
   s.dependency 'Amplify'
   s.dependency 'AmplifyPlugins/AWSDataStorePlugin'
-  s.dependency 'AmplifyPlugins/AWSAPIPlugin'
   s.platform = :ios, '13.0'
```

Comment:
Not related to this change, but our platform versions are different in `core` and `datastore`. We should align them.

Reply:
No, this is by design. Customers not using DataStore have no reason to have a min iOS 13 version.

Reply:
Ah ok, I didn't realize it was intentional.